backend/dal/IPimsRepository.cs

```diff
@@ -28,10 +28,14 @@ public interface IPimsRepository : IRepository
         #endregion
 
         #region Leases
-        ILeaseService Lease { get; }
+        ILeaseRepository Lease { get; }
         ILeaseTermRepository LeaseTerm { get; }
         #endregion
 
+        #region Deposits
+        ISecurityDepositRepository SecurityDeposit { get; }
```

Comment:
I think you forgot to add the securityDepositReturnRepository?

Reply:
odd, it should be there